app/tests/algorithms_tests/test_forms.py

```diff
@@ -1557,32 +1574,3 @@ def test_default_interface_for_algorithm_not_updated_when_adding_new_non_default
         assert new_io != io
         assert not new_iot.is_default
         assert old_iot.is_default
-
-    @pytest.mark.django_db
-    def test_is_default_is_updated_when_adding_an_already_existing_interface(
-        self,
-    ):
```

Comment:
With the new restriction, it is no longer possible to make a non-default interface the default interface of an algorithm through the create form. The user will need to delete the non-default interface first, and then add it again as default to achieve that. I think that is still a totally ok work-around, but this test here has to go.